Statistics/Distributions/Empirical.cs

```diff
@@ -102,24 +60,33 @@ public Empirical(double[] probabilities, double[] observationValues, bool probsA
             {
                 probabilityArray = probabilities;
             }
-
-            if (!IsMonotonicallyIncreasing(probabilityArray))
-            {   //TODO: sorting the arrays separately feels a little precarious 
-                //what if the user provides a non-monotonically increasing relationship?
-                //e.g. probs all increasing but values not or vice versa 
-                //I think we can probably do some checking where we sort only if both are not monotonically increasing?
-                Array.Sort(probabilityArray);
-            }
             CumulativeProbabilities = probabilityArray;
-            if (!IsMonotonicallyIncreasing(observationValues))
+            ObservationValues = observationValues;
+            //TODO: Should Min and Max be identified through extrapolation to the bounds of the probability domain?
+            Min = ObservationValues[0];
+            Max = ObservationValues[ObservationValues.Length - 1];
```

Comment:
This is going to rely on the sorting of observationValues. Somewhere we'll have to validate that, maybe in the viewmodel. We just need to make sure that the sorting we're expecting is consistent with the sorting we're enforcing

Reply:
good point. I could sort here. 

If I wanted to use paired data, we'd run into a bad situation where the model depends on statistics and statistics depends on the model?

Reply:
we dont want to depend on the model in statistics

Reply:
It looks like I handle the sorting problem in BuildFromProperties starting on line 97